src/components/ui/progress-bar.tsx

```diff
@@ -27,7 +32,7 @@ const ProgressBar: React.FC<ProgressBarProps> = ({ progress, label }) => {
         </div>
         <div className="text-right">
           <span className="text-xs font-semibold inline-block text-blue-600">
-            {`${progress.toFixed(2)}%`}
+            {usedDesc}
```

Comment:
How about:
```suggestion
            {usedDesc} ({progress.toFixed(2)}%)
```
We need to keep the percent progress as well, as `75%` threshold is significant and we don't want allocators to have to calculate it themselves in their heads.